invenio_rdm_records/resources/serializers/marcxml/schema.py

```diff
@@ -425,6 +426,22 @@ def get_rights(self, obj):
 
         return rights or missing
 
+    def get_license(self, obj):
+        """Get license."""
```

Comment:
Can you add a comment here, saying that the license is here and in the rights field for backward compatibility reasons (otherwise it will get removed in the future)